controller/src/beerocks/cli/beerocks_cli_bml.cpp

```diff
@@ -1307,6 +1321,53 @@ int cli_bml::get_dcs_scan_results_caller(int numOfArgs)
     return -1;
 }
 
+/**
+ * Caller function for client_get_client_list_caller.
+ *
+ * @param [in] numOfArgs Number of received arguments
+ * @return 0 on success.
+ */
+int cli_bml::client_get_client_list_caller(int numOfArgs)
+{
+    if (numOfArgs == 0) {
+        return client_get_client_list();
+    }
+    return -1;
+}
+
+/**
+ * Caller function for client_set_client_caller.
+ *
+ * @param [in] numOfArgs Number of received arguments
+ * @return 0 on success.
+ */
+int cli_bml::client_set_client_caller(int numOfArgs)
+{
+    if (numOfArgs == 2) {
+        return client_set_client(args.stringArgs[0], args.intArgs[1]);
+    } else if (numOfArgs == 3) {
```

Comment:
this implementation forces the first non-mac param to be provided if you wish to set the 2nd param.
this is not how you implement optional parameters.
cfr. `set_dcs_continuous_scan_params_caller` implementation

Reply:
Changed